plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchConfig.java

```diff
@@ -117,6 +124,72 @@ public ElasticsearchConfig setDefaultSchema(String defaultSchema)
         return this;
     }
 
+    @NotNull
+    public Duration getIndexMetaDataCacheTtl()
+    {
+        return indexMetaDataCacheTtl;
+    }
+
+    @Config("elasticsearch.index-metadata-cache-ttl")
+    public ElasticsearchConfig setIndexMetaDataCacheTtl(Duration indexMetaDataCacheTtl)
+    {
+        this.indexMetaDataCacheTtl = indexMetaDataCacheTtl;
+        return this;
+    }
+
+    @Min(0)
+    public long getIndexMetaDataCacheMaximumSize()
+    {
+        return indexMetaDataCacheMaximumSize;
+    }
+
+    @Config("elasticsearch.index-metadata-cache-maximum-size")
+    public ElasticsearchConfig setIndexMetaDataCacheMaximumSize(long indexMetaDataCacheMaximumSize)
+    {
+        this.indexMetaDataCacheMaximumSize = indexMetaDataCacheMaximumSize;
+        return this;
+    }
+
+    public boolean isUnionSchemaIndicesForAlias()
+    {
+        return unionSchemaIndicesForAlias;
+    }
+
+    @Config("elasticsearch.union-schema-indices-for-alias")
```

Comment:
This name is not very clear. I think we should rename this to "merge-alias-index-mappings"

Reply:
Done.